### ISSUE TITLE
Determine rules and the tech stack for all repositories

### ISSUE THREAD

**Issue by TymekGluch** [mztsApp/Docs#1]
# To do

- Create a documents and share with the rest of the organization containing information about the technology stack
- Define the rules of code creation, naming, styling, work flow, meetings and prepare the necessary prettier / eslint configurations
- Create repo / npm package for prettier / eslint configuration

## Required tech stack for all repositories:
React, Typescript, eslint, prettier, rtl, jest

## Tech stack per repositories::
1. UI - Linaria, StoryBook
2. Static Sites - Next.js, ?(Linaria | Styled-components | scss-modules)
3. App - ?Next.js, ?Firebase, ?(Linaria | Styled-components | scss-modules)
4. Site generator - Next.js, ?(Strapi | DatoCMS | WordPress | other headless CMS), ?Firebase, ?(Linaria | Styled-components | scss-modules)

**Comment by TymekGluch** [comment]
@GabrielGladykowski  `Property setting order in css ` in ```StylingApplication``` should contains also: transform-origin, justfy-items and align-content

**Comment by TymekGluch** [comment]
diffrence between align items and align content

The align-items property in CSS aligns elements within a container along the vertical axis. It can be used to align items to the top, bottom, or center of a container.

The align-content property, on the other hand, aligns the elements within a container along the vertical and horizontal axis, within a flexbox container. It can be used to distribute space between rows or columns of elements.

In short, align-items is used to align elements within a row or a single column and align-content is used to align elements within a grid and control the spacing between the rows or columns of a flex container

the same with justify content amd justify items
